lib/release-ops.js

```diff
@@ -319,15 +323,16 @@ function writeChangelog(releaseInfo) {
     (`\n${releaseInfo.rawChangelog}\n`).toEnd("CHANGELOG.tmp");
 
     // ensure there's a CHANGELOG.md file
-    if (!shelljs.test("-f", "CHANGELOG.md")) {
+    if (!test("-f", "CHANGELOG.md")) {
         fs.writeFileSync("CHANGELOG.md", "");
     }
 
     // switch-o change-o
-    fs.writeFileSync("CHANGELOG.md.tmp", shelljs.cat("CHANGELOG.tmp", "CHANGELOG.md"));
-    shelljs.rm("CHANGELOG.tmp");
-    shelljs.rm("CHANGELOG.md");
-    shelljs.mv("CHANGELOG.md.tmp", "CHANGELOG.md");
+    // `cat` returns a `ShellString`. Convert to string to write to file.
+    fs.writeFileSync("CHANGELOG.md.tmp", cat("CHANGELOG.tmp", "CHANGELOG.md").toString());
```

Comment:
`cat` returns a `ShellString` and `fs.writeFileSync` is throwing an error saying that this must be a `String`.

Reply:
Can you add that as a comment in the file?

Reply:
Done!